docs/assets/image-casa-module-diagram.drawio

```diff
@@ -0,0 +1 @@
+<mxfile host="Electron" modified="2022-07-18T11:08:33.103Z" agent="5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) draw.io/19.0.3 Chrome/102.0.5005.63 Electron/19.0.3 Safari/537.36" etag="tlzc22QVe07IabftvCVx" version="19.0.3" type="device"><diagram id="C63YB8BwHD-aRFYklcqd" name="Page-1">vVfJUuMwEP2aHEPZlu2QYzYCM0BRkwPD3ISl2CKKlJGVja8fyZZXOSmGCnCg3E+tlv1eL0oPTNaHuYCb5IEjTHuegw49MO153vDaVf81cMyBMAQ5EAuCcsipgAV5xznoFuiWIJw2HCXnVJKNAU38iDOGI9nAoBB833RbcooawAbGuBFdA4sIUmy5PRMkkxy99gYVfotJnBQnu+EwX3mF0SoWfMvMeT0PLLO/fHkNi1jm3DSBiO9rEJj1wERwLvOn9WGCqaa2SdvNidXyvQVm8iMbRvhXf7BEf0cvf3bPy/n+6dF97+sNOswO0q0h5AdkaYqZAhdY7LAwby+PBWPZN2Md1e2B8T4hEi82MNKre5UiCkvkmprlJWfSiO76yk6l4Cs84ZSLLBpwnQEYzrQnobSGj93p9MZReCwgIuobizXGmYo2Vm8miZJxREnMFCy5PhkaK1Ib1KuDsc2SIU7vx4caZFibY77GUhyVi1kFRUYfixQwiu6rfFF1kGNJI1eMIzQ5GpexK53Ug5HqP2TzbdmeuVilEkrC2RcrhsLXMAhtxVTqe1H0rdIMvJYywFbGcb5VmdBiHyPVcIzJhUx4zBmkswodV/rohK987rlmLlPlDUt5NLLAreRNzfCByN+15xcd6iow1vRgImfGsTCY+l69qe9cOQ4oELPV8Qug2p1Zje1PWBDFm5YzA9u5c1LilG9FhM/xaJSUUMRYnnP0c0fN8tmUEZiq6tg12/7l9QdWZY71kLp0H23V3nA0vcm6ZXd/xZcpN9d1m/XmD6x6c72Oegu/rNx8i+4JTKHFtRq+G/245miblVzRjSheSk2bIp6w+D6zpp7T1a/abW1NEGqVr1aKwldMx+XloDW2Pj0PS4VPzMNLDDo/aMobeLa8wLflDS4g708+ufOnq76zuHUe5nePszh2+q6l7pu6nfRVA0z66RfcTj5bVRbHHUqcpD1w/VZVhd9XVZ2027cLe6gxNNK3b10gFKYpiTKaoJA2XCO8OenKEVQatcF1ZvZ0dvzO5K+Pj9MZZktToz7oYL7APjxTzAlPnKgXLJX3B9dN5QdhM0Q+Js2u+t2+FSjwWoFAKzVyHqxAWXaUn92VMMqsfqLk7tXPQDD7Bw==</diagram></mxfile>
```

Comment:
Casa doesn't live within the Janssen Server . Please adjust the diagram to depict that.